addons/last-edit-tooltip/addon.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Last Edit Tooltip",
+  "description": "Hover over a project's share date for information on when it was last edited.",
+  "credits": [
+    {
+      "name": "TheColaber",
+      "link": "https://www.scratch.mit.edu/users/TheColaber"
+    }
+  ],
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["https://scratch.mit.edu/projects/*"]
+    }
+  ],
+  "tags": ["community", "recommended"],
```

Comment:
Nice try, but you can't make it `"recommended"` right away :P Please remove that tag, at least for now.